doc/contrib/release.rst

```diff
@@ -0,0 +1,148 @@
+.. _contribrelease:
```

Comment:
This isn't appropriate for the main documentation, because it is something that only one or two core devs .
I would prefer to keep it on the wiki, where we can update and maintain it without having to use PRs.